.github/workflows/ansible-test.yml

```diff
@@ -110,148 +110,13 @@ jobs:
           # https://github.com/ansible-community/ansible-test-gh-action#pull-request-change-detection
           pull-request-change-detection: false
 
-###
-# Unit tests (OPTIONAL)
```

Comment:
Does it mean that you removed the UT? there is not UT running at all after that PR?

Reply:
Well, I removed duplicated UT, there are already UT running in `github/workflows/test.yml`

Reply:
Maybe we can change the file names? 
ansible-test and test are confusing names... maybe ansible-sanity.yml?

Reply:
Yeah, that would be better.